_posts/2015-07-08-elasticsearch-is-coming.md

```diff
@@ -0,0 +1,798 @@
+---
+layout: post
+author: quentin_fayet
+title: "Elasticsearch Is Coming"
+excerpt: "Our first article about Elasticsearch, the full-text search engine based on Apache Lucene"
+modified: 2015-07-15
+tags: [gitlab, git]
+comments: true
+image:
+  feature: bandeau-gitlab.jpg
+  credit: Alban Pommeret
+  creditlink: http://reputationvip.io
+---
+
+# ELASTICSEARCH IS COMING
+
+## ROADMAP
+
+This article is the first of many to come about Elasticsearch. Elasticsearch is a vast and amazing technology, and I could go on talking about it for hours.
+
+So first, let me introduce the summary of this very first article:
+
+1. A smooth introduction to Elasticsearch
+2. Theoretical approach: What's under the armor of Elasticsearch ?
+3. Installation and configuration of Elasticsearch
+4. Querying: Basic CRUD
+5. Introduction to plugins
+
+***
+
+## How to read these articles ?
+
+Before we dive in Elasticsearch head first, let me tell you how I organized these articles.
+
+First of all, this first article is about the very basics of Elasticsearch, and I will not really talk about full-text research right now. The main reason is that I consider
```

Comment:
Well, because it's the pronoun I. Or maybe you saw an `I` written in uppercase that I didn't see ?